common/constants.js

```diff
@@ -475,5 +475,8 @@ module.exports = Object.freeze({
   SYSTEM_ERRORS: ['ECONNREFUSED', 'ECONNRESET', 'EPIPE', 'ETIMEDOUT', 'ESOCKETTIMEDOUT'],
   INTERNAL: 'internal',
   ALL: 'all',
-  DISABLED: 'disabled'
+  DISABLED: 'disabled',
+  REGEX_PATTERN: {
```

Comment:
A simple comment would mean ending up explaining the entire regex :-). There are many sites which do these easily. One such sample can be found [here](https://regex101.com/r/gHjwWN/1) Just copy paste your required regex and it will explain.